desktop/src/main/java/bisq/desktop/main/account/content/PaymentAccountsView.java

```diff
@@ -62,6 +62,8 @@ public void initialize() {
                 accountAgeWitnessService.getAccountAgeWitnessUtils().logSigners();
             } else if (Utilities.isCtrlShiftPressed(KeyCode.U, event)) {
                 accountAgeWitnessService.getAccountAgeWitnessUtils().logUnsignedSignerPubKeys();
+            } else if (Utilities.isAltOrCtrlPressed(KeyCode.C, event)) {
```

Comment:
Cmd+c is a very common key command, not sure if its better to use a more exotic one so that user do not trigger that accidentally. Prob. no harm by doing that, but confusing...

Reply:
As its only used for fiat, better move it to fiat class instead of base class

Reply:
I like adding it here, and I like ctrl+c since it's the common copy key and that's what we're doing here, copying the account info to clipboard.

Reply:
it opens a popup window right?

Reply:
or just copy to clipboard?

Reply:
sorry have not looked too close to the code base....

Reply:
Just copy to clipboard, no need for popup window.